packages/esm-patient-immunizations-app/src/immunizations/delete-immunization.modal.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { ModalHeader, ModalBody, ModalFooter, Button, InlineLoading } from '@carbon/react';
+import { useImmunizationsConceptSet } from '../hooks/useImmunizationsConceptSet';
+import { type ConfigObject } from '../config-schema';
+import { showSnackbar, useConfig } from '@openmrs/esm-framework';
+import { useTranslation } from 'react-i18next';
+import styles from './delete-immunization.scss';
+import { deletePatientImmunization } from '../hooks/useImmunizations';
+
+interface DeleteConfirmModelProps {
+  close: () => void;
+  handleDeleteDose: () => void;
```

Comment:
It looks like we're defining our own `handleDeletedose` function below so maybe we should delete this prop.